var/spack/repos/builtin/packages/opam/package.py

```diff
@@ -36,10 +41,18 @@ class Opam(AutotoolsPackage):
     depends_on("ocaml@:4.09.0", type="build", when="@:1.2.2")
     depends_on("ocaml", type="build", when="@2.0.0:")
 
-    parallel = False
+    # TODO: Is this really the correct constraint? I can't find any issues with
+    # building newer opams in parallel, and the constraint was added in the
+    # package's first commit with no message.
+    with when("@:2.0.0"):
+        parallel = False
```

Comment:
`make` is a frontend to the `dune` builder, which is parallel. So there is no advantage to run `make` in parallel.
Can you please add this as a comment? Thanks ;-)

Reply:
I found this thread:
https://github.com/ocaml/opam/issues/3585
So in my opinion, it is prefereable to leave `parallel = False`

Reply:
Thank you for the context! I'll add a comment.

Reply:
Thanks, this comment can be marked as resolved